PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -1576,6 +1638,63 @@ struct HfCandidateCreatorXic0Omegac0 {
       registry.fill(HIST("hKfXiC0_ldl"), kfXic0Candidate.ldlXic);
       registry.fill(HIST("hDcaXYCascadeToPVKf"), kfXic0Candidate.kfDcaXYCascToPv);
 
+      // QA
+      kfXic0Qa.v0DauPosX = kfPos.GetX();
```

Comment:
why filling first the struct and then the table? Can't you directly fill the table?